src/components/Wallets/WalletPage.vue

```diff
@@ -39,21 +39,21 @@ export default {
   created () {
     this.getAccountAssets()
   },
-  methods: { 
+  methods: {
     ...mapActions([
       'getAccountAssets'
     ]),
     openDefaultWallet () {
       let walletId = this.$route.params.walletId
 
-      if (walletId == 'undefined' || walletId == null) {
+      if (walletId === 'undefined' || walletId == null) {
```

Comment:
‘walletId === 'undefined'’ will check equality to string “undefined”. Is it what you check?

If not you can use just ‘if (walletId)’

Reply:
How about this 
if (isUndefined(walletId) || walletId == null)